packages/core/src/styles/_chart-holder.scss

```diff
@@ -7,32 +7,24 @@ div.chart-holder {
 /////////////////////////////////////////////
 ///          Theme styles                ////
 /////////////////////////////////////////////
-.carbon--theme--white {
-	&.chart-holder {
-		background-color: $ui-background;
-	}
+.carbon--theme--white.chart-holder {
+	background-color: $ui-background;
```

Comment:
Quick question - if the token is `$ui-background` do we need the `@include carbon--theme` and `.carbon--theme--x` namespacing? could we instead just namespace the chart holder like `#{$prefix}--chart-holder` and use the token?

Reply:
yeah, this has been done